draft-ietf-quic-qlog-quic-events.md

```diff
@@ -1300,8 +1300,8 @@ class IPAddress : string | bytes;
 enum PacketType {
     initial,
     handshake,
-    zerortt = "0RTT",
-    onertt = "1RTT",
+    0RTT,
```

Comment:
nit: one could argue that RTT should be lowercase for consistency with the other types.

Reply:
Somewhat disagree. RTT is an acronym and the others are not. The RFCs also use the uppercased version of this. Finally, for backwards-compat with existing qlog implementations, 0RTT and 1RTT is easiest.

Reply:
@rmarx sounds good. 
Are there any other concerns, or can this be merged?

Reply:
This is fine I think. 

For other, larger changes, we'd ideally also add an entry in the changelist on the bottom. But this is such a small thing that it's not needed.